LXReorderableCollectionViewFlowLayout/LXReorderableCollectionViewFlowLayout.m

```diff
@@ -50,8 +50,9 @@ - (UIView *)LX_snapshotView;
 @implementation UICollectionViewCell (LXReorderableCollectionViewFlowLayout)
 
 - (UIView *)LX_snapshotView {
+    
     if ([self respondsToSelector:@selector(snapshotViewAfterScreenUpdates:)]) {
-        return [self snapshotViewAfterScreenUpdates:YES];
+        return [self snapshotViewAfterScreenUpdates:NO];
```

Comment:
Seems that it has to do with a bug on iPhone 6/6+'s when zooming the display (running an app that is not optimized for the larger screen size).

http://stackoverflow.com/q/25873234/1959707